### PR TITLE
Task 1

### PR DESCRIPTION
**Pull request by deser**
Kavabanga

Task: https://epa.ms/nodejs18-hw1

### CODE REVIEW

File: lint-staged.config.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '*.ts': ["prettier --write ./src/**/*.ts", "tslint --fix", "git add"],
```
Comment (deser) [inline lint-staged.config.js:2]: are you sure that always fixing is good idea?

File: src/models/User.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export default class User {
+    constructor() {
+        console.log('User module');
```
Comment (deser) [inline src/models/User.ts:3]: it's better to use logging abstraction like https://www.npmjs.com/package/winston

**Comment by deser** [comment]
tets

File: src/models/User.ts (hunk 2 above)
Comment (sleeping-cucumber) [reply src/models/User.ts:3]: I guess for this task we need this logs just to check that server is running, not for real logging

File: lint-staged.config.js (hunk 1 above)
Comment (sleeping-cucumber) [reply lint-staged.config.js:2]: Hm, probably better do manually, will fix

**Pull request merged**